docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -94,61 +94,90 @@ expands on what differentiates IBM, including microsimulation, from other types
+which allow relapse hazard rates to vary over time since treatment line initiation. Read more about how we calculated relapse hazard rates :ref:`here <calc_mortality_relapse_hazard>`.
+
+.. todo:: 
+
+    Elaborate on example from Multiple Myeloma model
 
 Incorporating uncertainty
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Our microsimulation framework is formulated to routinely propagate forward the uncertainty captured by these draws. Monte Carlo methods allow us to effectively incorporate different kinds of uncertainty into our models. 
+Microsimulation methods provide a useful approach to modeling stochastic uncertainty in a conceptually valid manner because chance – that is, reliance on randomly generated numbers – 
+is used to govern movement between health states for individual simulants from a single computer-generated population, as opposed to comparing the role of chance across multiple 
+populations with differing characteristics.
 
-Monte Carlo microsimulation methods provide a useful approach to modeling stochastic uncertainty in a conceptually valid manner because chance – that is, reliance on randomly generated numbers – is used to govern movement between health states for individual simulants from a single computer-generated population, as opposed to comparing the role of chance across multiple populations with differing characteristics.
+In the context of healthcare estimates typically characterized by considerable variance across quantities of interest, the use of Monte Carlo methods to run simulations using iterative 
+draws from probability distributions provides an effective means of incorporating parameter uncertainty. 
 
-In the context of healthcare estimates typically characterized by considerable variance across quantities of interest, the use of Monte Carlo methods to run simulations using iterative draws from probability distributions provides an effective means of incorporating parameter uncertainty. Simulation methods have the added advantage of reducing confounding variance between different scenarios by ensuring that the simulated population is identical across all scenarios analyzed. The only difference is the presence or absence of health interventions. As a further refinement of these methods, microsimulations represent each person in the population as an individual simulant with attributes variably assigned across simulants to reflect the population of interest. By incorporating heterogeneity at the level of the individual simulant, microsimulations are able to provide an additional degree of flexibility and detail (such as stratifying results according to demographic or epidemiological attributes) not available in population-level models. 
+Simulation methods have the added advantage of reducing confounding variance between different scenarios by ensuring that the simulated population is identical across all scenarios analyzed. 
```

Comment:
I think this, and the first paragraph of this section, are really describing common random numbers (CRN), not Monte Carlo methods in general. If I'm correctly following the purpose of this section, it is to compare microsimulation methods with non-microsimulation methods; and then later, we will compare Vivarium with competing microsimulation tools. Is that right? If so, CRN is a _Vivarium_ feature that distinguishes it from (most) other microsimulation tools, not an aspect of microsimulation that distinguishes it from non-microsimulation methods.

Overall, I'm not sure I'm convinced there is a key advantage to note here: it seems like non-microsim methods can use Monte Carlo approaches to uncertainty as well.